_data/anish9701/ANISH.md

```diff
@@ -0,0 +1,7 @@
+---
+name: V A S S Sai Anish Kumar 
+institution: Geethanjali college of engineering and technology
+profile_pic: anish.jpg

```

Comment:
Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `anish.jpg` to `anish.JPG ` in the markdown file.
2. Rename `anish.JPG ` to `anish.jpg`.
I'd suggest you rename the file!